packages/Interface/test/tstKokkosView.cpp

```diff
@@ -210,14 +210,60 @@ TEUCHOS_UNIT_TEST_TEMPLATE_2_DECL( View, basic_reduce_kernel, Scalar,
     typename ViewType::HostMirror host_data =
         Kokkos::create_mirror_view( data );
     Kokkos::deep_copy( host_data, data );
-    double test_sum = 0.0;
+    Scalar test_sum = 0.0;
     for ( int i = 0; i < size; ++i )
     {
         test_sum += host_data( i );
     }
     TEST_EQUALITY( test_sum, sum );
 }
 
+//---------------------------------------------------------------------------//
+// Test for creating a view and running a basic scan kernel
+TEUCHOS_UNIT_TEST_TEMPLATE_2_DECL( View, basic_scan_kernel, Scalar, DeviceType )
+{
+    // Get types.
+    using ExecutionSpace = typename DeviceType::execution_space;
+    using MemorySpace = typename DeviceType::memory_space;
+
+    // Create a view in the execution space.
+    using ViewType = Kokkos::View<Scalar *, MemorySpace>;
+    const int size = 1000;
+    ViewType data( "data", size );
+    ViewType result( "result", size );
+
+    // Populate the view in the execution space.
+    Kokkos::parallel_for( Kokkos::RangePolicy<ExecutionSpace>( 0, size ),
+                          FillFunctor<ViewType>( data ) );
```

Comment:
`FillFunctor` is not a good name. I was going to tell you to use `Kokkos::deep_copy()` and then Iooked at the implementation and I realized you meant `iota( data )`

Reply:
going to get rid of this test so N/A